### PR TITLE
INGM-411: make new example about position

### PR DESCRIPTION
**Pull request by WoodElv**
### Description

This example move a position motor 3 times with a constant velocity.
To do that, the example follows the steps below:
1. Connect a drive via EtherCAT-CoE.
2. Select the operation mode in profile position, that is, position mode and trapezoidal profile.
3. According to [this documentation](https://drives.novantamotion.com/summit/position-modes-csp-pp-ip-p#Positionmodes(CSP,PP,IP,P)-Profileposition(PP)) Max. profile acceleration, Max. profile deceleration and Max. profile velocity have to be configured.
4. Enable the motor.
5. Run a position ramp 3 times.
6. Disable the motor.
7. Disconnect the drive. 

Fixes # INGM-411

### Type of change

Please add a description and delete options that are not relevant.

- [x] Position ramp example.

### Tests
- [x] Unit test for position_ramp example.
- [x] Run tests.

### Documentation

Please update the documentation.

- [x] Update docstrings of every function, method or class that change.
- [x] Use [type hints](https://docs.python.org/3/library/typing.html) for every function and argument.
- [x] Build documentation locally to verify changes.

### Code formatting

- [x] Use black package to format the code: `black -l 100 ingeniamotion tests`. It is recommended to configure the code editor to automatically format the code using black with a max length line of 100.

### CODE REVIEW

File: examples/position_ramp.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    """
+    Creates a position ramp from current position to ``final_position`` as a ramp slope.
+
+    Args:
+        final_position: target position in counts.
+        mc: Controller with all the functions needed to perform a position ramp.
+    """
+    done = False
+    mc.motion.move_to_position(final_position)
+    while not done:
```
Comment (pzinemanas) [inline examples/position_ramp.py:15]: This while will iterate only once, right? Am I missing something?

File: examples/position_ramp.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+from ingeniamotion import MotionController
+from ingeniamotion.enums import OperationMode
+
+
+def position_ramp(final_position, mc: MotionController) -> None:
+    """
+    Creates a position ramp from current position to ``final_position`` as a ramp slope.
```
Comment (pzinemanas) [inline examples/position_ramp.py:7]: It will only create a ramp if the operation mode is set to PROFILE_POSITION

File: examples/position_ramp.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    mc.configuration.set_max_profile_acceleration(max_acceleration_deceleration)
+    mc.configuration.set_max_profile_deceleration(max_acceleration_deceleration)
+    # Enable the motor
+    mc.motion.motor_enable()
+
+    target_positions = [1500, 3000, 0]
+    for current_target in target_positions:
+        position_ramp(current_target, mc)
+        actual_position = mc.motion.get_actual_position()
+        print(f"Actual final position: {actual_position}")
```
Comment (pzinemanas) [inline examples/position_ramp.py:43]: Actual or final?

File: examples/position_ramp.py (hunk 1 above)
Comment (WoodElv) [reply examples/position_ramp.py:15]: No, you are right. I'm going to change it. Thanks.

File: examples/position_ramp.py (hunk 2 above)
Comment (WoodElv) [reply examples/position_ramp.py:7]: I'm going to delete this function.

File: examples/position_ramp.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        print(f"Actual position: {actual_position}")
+
+    # Disable the motor
+    mc.motion.motor_disable()
+
+    mc.communication.disconnect()
+
+
+if __name__ == "__main__":
+    # Before running this example, setting-up the drive with a connected motor is required
```
Comment (MartinALL91) [inline examples/position_ramp.py:37]: "make sure the drive is properly configured and connected to a motor" or something like that

**Review by GullinBustin — APPROVED** [review]

**Review by pzinemanas — APPROVED** [review]

**Review by MartinALL91 — APPROVED** [review]

**Pull request merged**